generate-client.sh

```diff
@@ -30,9 +30,26 @@ then
 fi
 echo "[SUCCESS]"
 
+echo "Cloning source code repository from GitHub..."
+git clone --depth 1 --single-branch --branch staging $REPO_URL /repo
 
-echo "Creating package.json from template..."
+echo "Searching for models files..."
+cd /repo/src
+globs=$(find . -iname '*.models.ts')
```

Comment:
```suggestion
globs=$(find . -iname '*.public-models.ts')
```
Maybe? Just to make it extra explicit?